.gitignore

```diff
@@ -1,3 +1,7 @@
+.env
```

Comment:
Can these be removed? `.env` is already listed in this file further down. If you can migrate your data from `.env.botplays` and `.env.copy`, that would be awesome.

Reply:
done